app/models/custom_field.rb

```diff
@@ -47,6 +47,13 @@ class CustomField < ActiveRecord::Base
 
   DATA_TYPES = %i(string text number range group boolean)
 
+  def self.loan_questions(field_set = nil)
+    field_set &&= "loan_#{field_set}"
+    field_set ||= ['loan_criteria', 'loan_post_analysis']
```

Comment:
Yeah, it's a little cutesie. `field_set` is a string, either 'criteria' or 'post_analysis', or nil. If it's given, it needs to be prepended, and if it's not, it needs to be set to both. What if I just add this as a comment?